llama_stack/apis/post_training/post_training.py

```diff
@@ -139,7 +132,7 @@ class PostTrainingJobStatusResponse(BaseModel):
     """Status of a finetuning job."""
 
     job_uuid: str
-    status: PostTrainingJobStatus
+    status: JobStatus
```

Comment:
did you not need to regenerate the openapi spec etc. to apply these changes to the client sdk?

Reply:
Currently, I manually changed the client SDK to accommodate the server side API change and test. Will use openapi spec to generate when we finalize the server side APIs